backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -646,4 +648,10 @@ object VeloxConfig {
       .internal()
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("10MB")
+
+  val VELOX_MAX_COMPILED_REGEXES =
+    buildConf("spark.gluten.sql.columnar.backend.velox.maxCompiledRegexes")
+      .doc("Controls maximum number of compiled regular expression patterns per batch.")
```

Comment:
Velox's comments for the corresponding looks clearer. Maybe, we can just copy them here.
https://github.com/facebookincubator/velox/blob/8875b4d08b26be2743bb5e79efe0548bebd08042/velox/core/QueryConfig.h#L91-L92